### ISSUE TITLE
Support BrainVoyager srf files

### ISSUE THREAD

**Issue by dfsp-spirit** [dfsp-spirit/freesurferformats#17]
We could support BrainVoyager srf file format:
See https://support.brainvoyager.com/brainvoyager/automation-development/84-file-formats/344-users-guide-2-3-the-format-of-srf-files

**Comment by dfsp-spirit** [comment]
There are some demo files available here: https://brainvoyager.com/bv/sampledata/index.html

I did not find the license yet, we will need to find out whether we are allowed to ship them for unit tests or whether we will have to re-download live each time.

**Comment by dfsp-spirit** [comment]
The meshes are now supported. For this to really make sense, support for surface data would be great. Here is an example thickness surface overlay file in SMP format. I created the file in the EDU version for the demo CG_LHRH_D65534.srf mesh.

[CG_LHRH_D65534_Thickness.zip](https://github.com/dfsp-spirit/freesurferformats/files/5075121/CG_LHRH_D65534_Thickness.zip)

**Comment by dfsp-spirit** [comment]
I received no reply regarding the license for the meshes, but I think the file name allows them to be attributed to a person, so we will definitely not ship them. Now that we have a mesh writer for SRF, this is not needed anymore anyways: we can convert some existing mesh to the format ourselves.

The website linked above only holds specs for older file format versions than the ones used in the current Brainvoyager release, as I noticed (e.g., for SMP format, website has specs for v2 and v3, but BV writes v5 files, and it seems they are **not** compatible). I wrote to support to request new specs, but the fact alone that versions change so quickly (the specs are from 2018) leaves me wondering whether supporting them is worth it. We'll see. UPDATE: Support answered, specs will become available later this year. (There are alternatives using Plugins inside BV that allow reading the files in custom code via BV functions, but in our case that does not help of course.)
